fdmt/cpu_fdmt.py

```diff
@@ -4,7 +4,8 @@
 import numpy as np
 from time import time
 from attr import attrs, attrib, cmp_using
-
+#from . import fdmt_iter_par
+from .fdmt_njit import fdmt_iter_par
```

Comment:
I think we favour absolute imports.

Was there any rationale behind moving that function to another folder? (Not saying it's wrong, just wondering if it some numba requirement.)

Don't merge just yet, I want to have quick look as well.

Reply:
Yeah, was gonna say, but since pre-commit isn't on here yet, I thought it was understandable, there's probably many things not formatted correctly here yet, this being a more obvious one.

@larskuenkel I told Robert but should've posted publicly (probably on my project board view), I'm currently rewiring some of the common files, like Precommit's, on sps-ops, which will serve as the central place for those global files. I have a script there that'll push those changes to every repo, including this one, so we can merge this, and I'll have the pre-commit in another commit to main after.

Reply:
I needed at least a wrapper function for numba, but it could be in this file or a separate one (as is currently).  Separating it just made it clear in my head, and easier to develop.  Is there a preference either way?

Reply:
I think the current layout with having class functions in one file and other functions in another is good.